_lessons/17-adding-placeholder-text.md

```diff
@@ -0,0 +1,196 @@
+---
+title: Adding Placeholder Text
+lesson: 17
+layout: default
+---
+
+Now, we are ready to begin filling in the layout with some placeholder text. First, let's add some headings to the header. In the last section, we learned that the tags `h1` through `h6` are used for headings. We'll use `h1` for the biggest title, and `h2` for a subtitle. 
+
+```
+<div id="header">
+  <h1>My First Webpage</h1>
+  <h2>My Web Development Journey Begins</h2>
+</div>
+```
+
+Next, let's add a few fake links to the sidebar using the `a` tag, which stands for anchor tag. Eventually, we can use these to link to more pages in our website, or other sites we like. For now, they will be placeholders so that we can style them. 
+
+The `href` attribute is used to specify which URL it links to. It's important to set this to something in our placeholders, because links that don't go anywhere will actually look different on the page. We want to see what links that go somewhere look like. 
+
+Since these are placeholders, one trick we can do is to set these to `"#"`, which is a way of linking to the current page. 
+
+```
+<div id="sidebar">
+  <a href="#">Link One</a>
+  <a href="#">Link Two</a>
+  <a href="#">Link Three</a>
+</div>
+```
+
+Next up is our content, which we can fill in with Lorem Ipsum text. There are many Lorem Ipsum generators online, and if you want to have some fun with it, there are even themed ones online that generate text about cupcakes, zombies, bacon and more. For this example, I'll just use the standard [Lorem Ipsum Generator](https://www.lipsum.com/) to generate 3 paragraphs of text.
```

Comment:
Believe it or not, a lot of people don't actually know what "Lorem Ipsum" is.
```suggestion
Next up is our content, which we can fill in with placeholder text. The standard text that most people use for placeholder purposes is called "Lorem Ipsum", which is basically just a bunch of fake Latin that looks like it might be real text, but doesn't actually mean anything. There are many Lorem Ipsum generators online, and if you want to have some fun with it, there are even themed ones online that generate text about cupcakes, zombies, bacon and more. For this example, I'll just use the standard [Lorem Ipsum Generator](https://www.lipsum.com/) to generate 3 paragraphs of fake text.
```